package.json

```diff
@@ -50,6 +50,35 @@
         ],
         "default": "default"
       },
+      {
+        "name": "Token name format",
+        "type": "picker",
+        "id": "tokenNameFormat",
+        "submenu": true,
+        "options": [
+          {
+            "name": "Constant Case",
+            "value": "constant"
+          },
+          {
+            "name": "Snake Case",
+            "value": "snake"
+          },
+          {
+            "name": "Camel Case",
+            "value": "camel"
+          },
+          {
+            "name": "Pascal Case",
+            "value": "pascal"
+          },
+          {
+            "name": "Do not format",
+            "value": "none"
+          }
+        ],
+        "default": "constant"
```

Comment:
I added "constant case" as default.  Since it's sort of best practice. However, it's a breaking change. We can use "none" instead. WDYT @zeplin/dx @mertkahyaoglu?

Reply:
I think we can keep "constant case" as default. If it happens to be confusing for users, we can change it to the "none" then.